src/styles/reset.scss

```diff
@@ -0,0 +1,19 @@
+html {
+  font-family: $catalog-font-family;
+  font-weight: $catalog-font-weght;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
move to the normalize file
also, body doesn't have paddings & border
```suggestion
body {
  margin: 0;
}
```

Reply:
fix it please
![image](https://user-images.githubusercontent.com/98764641/224283919-b4691284-aaf2-4cf5-bdaf-44eb1fc5ea8c.png)

Reply:
not fixed